ui.tests/test-module/specs/LocaleTest.spec.js

```diff
@@ -19,14 +19,32 @@ const dataSet = require('../../test-module/libs/commons/localeDataSets');
 const baseUrl = '/content/dam/formsanddocuments/core-components-it/samples/af2-form-translation/jcr:content?wcmmode=disabled',
     languageUrl = baseUrl + '&afAcceptLang=';
 
+const waitForLangDataToLoad= () => {
+    cy.document().then((doc) => {
+        return new Cypress.Promise((resolve, reject) => {
+            let ready = false;
+            doc.addEventListener('AF_LanguageInitialised', () => {
```

Comment:
You need to ensure that the event listener is attached before the event is triggered. Thats why the test cases are failing, else the PR LGTM

Reply:
This is happening only for the locale added via overlay, i was guessing there is some issue with that